lib/engine/game/g_18_al.rb

```diff
@@ -206,9 +238,10 @@ def tile_lays(entity)
       private
 
       def route_bonus(route, type)
```

Comment:
I think I may have made a mistake here when I rebased. This will probably bork if there are multiple abilities.